modular-docs-manual/content/topics/module_nesting-assemblies.adoc

```diff
@@ -20,14 +20,15 @@ To solve this problem, restore the `:context:` variable to its previous value wh
 +
 [source,asciidoc]
 ----
-:parent-context: {context}
+ifdef::context[:parent-context: {context}]
 ----
 
-. Add the following line at the end of your assemblies to restore the saved context:
+. Add the following line at the end of your assemblies to restore, if one already existed, the saved context:
```

Comment:
This sentence is a little out of order. Can we move this qualifying phrase to the end of the rest of the sentence?

Reply:
ACK.

Reply:
updating - please re-review.